src/types/gqlEnums.ts

```diff
@@ -75,18 +75,18 @@ export const HmisEnums = {
     OTHER: 'Other',
   },
   BoundType: { MAX: 'MAX', MIN: 'MIN' },
-  CeOpportunityStatus: { closed: 'closed', locked: 'locked', open: 'open' },
+  CeOpportunityStatus: { closed: 'Closed', locked: 'Locked', open: 'Open' },
   CeReferralStatus: {
-    accepted: 'accepted',
-    in_progress: 'in_progress',
-    initialized: 'initialized',
-    rejected: 'rejected',
+    accepted: 'Accepted',
+    in_progress: 'In progress',
+    initialized: 'Initialized',
+    rejected: 'Rejected',
```

Comment:
Just noting the discrepancy between these and the `ReferralStatusChip`. If we are using these as labels in some places, it will be inconsistent.

Reply:
agreed, thanks!